src/clj/rems/api/applications.clj

```diff
@@ -140,7 +143,8 @@
       :roles #{:logged-in}
       :return [schema/ApplicationOverview]
       :query-params [{query :- (describe s/Str "search query [documentation](https://github.com/CSCfi/rems/blob/master/docs/search.md)") nil}]
-      (ok (->> (applications/get-my-applications (getx-user-id))
+      (ok (->> (application/get-full-personalized-applications-by-user (getx-user-id))
+               (mapv project-application-overview)
                (filter-with-search query))))))
```

Comment:
We can change the order of these operations like in the next function (slightly improve perf)